(#862) Add documentation of duplicating a Group (#872)
chocolatey/docs: input/en-us/central-management/usage/website/groups.md

Similar to the [Edit Group](#editing-a-group) action, select the :gear: **Actions** button on the left-hand side of the group, and instead select **Delete**.
You will be prompted to confirm the deletion.

## Duplicating a Group

> :choco-info: **NOTE**
>
> If you do not see the **Duplicate** menu entry or the :gear: **Actions** buttons, consult your administrator to determine if you have the appropriate role assigned to your account.

On the main Groups page, find the Group you want to duplicate.
You can enter a search term in the text field to filter results by typing in part of their Name or Description and clicking the :mag: button.

> Similar to the [Edit Group](#editing-a-group) action, select the :gear: **Actions** button on the left-hand side of the group, and instead select **Duplicate**.

Reviewer: ```suggestion
Similar to the [Edit Group](#editing-a-group) action, select the :gear: **Actions** button on the left-hand side of the Group, and instead select **Duplicate**.
```

> If the Group being duplicated is a system managed Group, a confirmation message is shown about the duplicated Group will not be system managed.

Reviewer: ```suggestion
If the Group being duplicated is a system managed Group, a confirmation message will be shown indicating that the duplicated Group will not be system managed.
```

> A success message will be shown that mentions that the group was duplicated will be shown, and if you have permissions to Edit a Group a Dialog to Edit this duplicated Group will be shown.

Reviewer: ```suggestion
A success message will be shown that mentions that the Group was duplicated, and if you have permissions to Edit a Group a dialog to edit this duplicated Group will be shown.
```
